### ISSUE TITLE
SUDDEN FAILURES

### ISSUE THREAD

**Issue by vladislav-prishchepa** [OPEXGroup/ITCC.Library#44]
```
[16.02.2017 21:59:48.587] [DEBUG] [THR 4     ] [    MAIL LOG] Email logger starting
[16.02.2017 21:59:49.259] [ INFO] [THR 4     ] [        MAIN] Loggers initialized: File, SystemEvent, Email, Request
[16.02.2017 21:59:53.029] [ INFO] [THR 4     ] [CONFIGURATION] Using 4 MS Word processes
[16.02.2017 21:59:53.780] [DEBUG] [THR 4     ] [     DB LOAD] Account already exists
[16.02.2017 21:59:53.795] [DEBUG] [THR 4     ] [     DB LOAD] All questions already exist
[16.02.2017 21:59:53.857] [DEBUG] [THR 4     ] [     DB LOAD] All questions already exist
[16.02.2017 21:59:53.857] [ INFO] [THR 4     ] [          DB] Initial data loaded
[16.02.2017 21:59:53.857] [ INFO] [THR 4     ] [        MAIN] Email controller configured
[16.02.2017 21:59:53.951] [ INFO] [THR 4     ] [        MAIN] Resend controller configured
[16.02.2017 21:59:53.967] [ INFO] [THR 4     ] [HTTP CONTROL] Using encrypted connections
[16.02.2017 21:59:55.127] [ CRIT] [THR 4     ] [ HTTP SERVER] EXCEPTION (InvalidOperationException)
Instance 'WmiApSrv' does not exist in the specified Category.
   at System.Diagnostics.CounterDefinitionSample.GetInstanceValue(String instanceName)
   at System.Diagnostics.PerformanceCounter.NextSample()
   at System.Diagnostics.PerformanceCounter.get_RawValue()
   at ITCC.HTTP.Server.Service.ServerStatistics`1.GetProcessInstanceName(Int32 pid) in C:\EsdDemo\Repository\lib\ITCC.Library\ITCC.HTTP.Server\Service\ServerStatistics.cs:line 543
   at ITCC.HTTP.Server.Service.ServerStatistics`1.GetCurrentProcessInstanceName() in C:\EsdDemo\Repository\lib\ITCC.Library\ITCC.HTTP.Server\Service\ServerStatistics.cs:line 530
   at ITCC.HTTP.Server.Service.ServerStatistics`1..ctor(Int32 criticalMemoryValue, MemoryAlarmStrategy memoryAlarmStrategy) in C:\EsdDemo\Repository\lib\ITCC.Library\ITCC.HTTP.Server\Service\ServerStatistics.cs:line 55
   at ITCC.HTTP.Server.Core.StaticServer`1.PrepareStatistics(HttpServerConfiguration`1 configuration) in C:\EsdDemo\Repository\lib\ITCC.Library\ITCC.HTTP.Server\Core\StaticServer.cs:line 118
   at ITCC.HTTP.Server.Core.StaticServer`1.Start(HttpServerConfiguration`1 configuration) in C:\EsdDemo\Repository\lib\ITCC.Library\ITCC.HTTP.Server\Core\StaticServer.cs:line 61

[16.02.2017 21:59:55.127] [DEBUG] [THR 4     ] [    MAIL LOG] Sending email with 1 entries
[16.02.2017 21:59:55.158] [ CRIT] [THR 4     ] [        MAIN] EXCEPTION (NullReferenceException)
Object reference not set to an instance of an object.
   at ITCC.HTTP.Server.Core.StaticServer`1.Start(HttpServerConfiguration`1 configuration) in C:\EsdDemo\Repository\lib\ITCC.Library\ITCC.HTTP.Server\Core\StaticServer.cs:line 104
   at EsdDemo.Server.Core.Controllers.HttpServerController.Start() in C:\EsdDemo\Repository\src\EsdDemo.Server.Core\Controllers\HttpServerController.cs:line 45
   at EsdDemo.Server.Core.Common.ServerApplication.StartControllers() in C:\EsdDemo\Repository\src\EsdDemo.Server.Core\Common\ServerApplication.cs:line 81

[16.02.2017 21:59:55.158] [ INFO] [THR 4     ] [        MAIN] EsdDemo service has started successfully

```

**Comment by VladimirTyrin** [comment]
Second exception is fixed, will be included in next version

**Comment by VladimirTyrin** [comment]
Fixed in v2.8.2
